components/camel-hyperledger-aries/src/main/java/org/apache/camel/component/aries/Constants.java

```diff
@@ -0,0 +1,44 @@
+/*-
+ * #%L
```

Comment:
This section should be removed from the license header.

Reply:
These headers are generated with the maven plugin. I suppose you don't manually maintain license headers across all files. I can remove all headers with plugin and perhaps then regenerate in the way you want it, but how would I do that?